### ISSUE TITLE
neutral new tab background for dark mode/no background images

### ISSUE THREAD

**Issue by rebron** [brave/brave-browser#8061]
## Description 
In dark mode, the new tab page no background image looks to be too bright for those wanting to be in "dark mode". Maintain current gradient background for light theme but use a different background for dark theme.

## Actual result:
<img width="1277" alt="Screen Shot 2020-02-04 at 2 24 20 PM" src="https://user-images.githubusercontent.com/5951041/73793135-9b847780-475a-11ea-886b-60625aac8a39.png">

## Expected result:
Design to be added here. @karenkliu

**Comment by washere** [comment]
@rebron Thanks for the reply.

*** NO "dark" designs**, as you put it, is not what many online (github, reddit, forums etc etc) want.
*** Nor "dark photos"** etc. as many online have said.

Many of us just want to be able to:

**SET THE NTP BACKGROUND TO BLACK**

* Other colors would be a luxury right now. Just like Windows Background option to set to a solid color. 

WE JUST WANT TO HAVE BLACK BACKGROUND & BE ABLE TO REMOVE OTHER "DESIGN WIDGETS" TOO. Simple, should take a few minutes to code.

WE DO NOT WANT DARK DESIGNS.
JUST PLAIN BLACK SOLID COLOR COLOR & NO BELLS OR WHISTLES WIDGETS if we choose to remove them. Like a minimal Morpheon Theme design:

![image](https://user-images.githubusercontent.com/5448047/73799086-cac3d480-47ac-11ea-8f5e-ed601d49505b.png)

Next step would be the default Search_engine Text_Field:

![image](https://user-images.githubusercontent.com/5448047/73799112-e7f8a300-47ac-11ea-9fbd-28ec615e8ccc.png)

THANK YOU VERY MUCH for you taking up this CRITICAL issue for many influential users online.

@rebron you are very much appreciated by many.

**Comment by aaronsantamaria** [comment]
+1 Community
https://community.brave.com/t/garish-pink-color-on-new-tab-page/167206

**Comment by crisc82** [comment]
Yes, please add other colors to options. Preferably some colors/gradients to also match with dark theme.
Personally I would prefer a dark grey-light gray gradient.

For the moment I use this extension https://chrome.google.com/webstore/detail/empty-new-tab-page-dark-g/penfkpiinoeikpfhnmgfogeodpomnncg
